cli/parse-options.js

```diff
@@ -42,7 +42,7 @@ function parseOptions (options, next) {
 
   if (options.dbUrl) {
     if (!urlParse(options.db.url).auth) {
-      return next(new Error('Authentication details missing from database URL: ' + options.db.url))
+      throw new Error(new Error('Authentication details missing from database URL: ' + options.db.url))
```

Comment:
it’s `throw new Error('Authentication details missing from database URL: ' + options.db.url)`. The `Error` Constructor only accepts a string as argument